backend/src/main/java/feedzupzup/backend/feedback/domain/PostedAt.java

```diff
@@ -0,0 +1,36 @@
+package feedzupzup.backend.feedback.domain;
+
+import jakarta.persistence.Embeddable;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@Embeddable
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class PostedAt {
+
+    private static final String TIME_ZONE_AREA = "Asia/Seoul";
+
+    private LocalDateTime postedAt;
```

Comment:
p2)
기존 1개의 변수만 포장했던 값 객체는 value라고 표현해와서, value로 바꿔주고 '@Column'으로 컬럼명을 명시해주는 건 어떤가요?
```suggestion
    @Column(name = "posted_at")
    private LocalDateTime value;
```